CMakeLists.txt

```diff
@@ -24,7 +24,17 @@ else()
     get_filename_component(CLANG_EXE_DIR ${CLANG_EXE} DIRECTORY)
     find_package(AMDDeviceLibs PATHS /opt/rocm)
     set(CLANG_BIN "${CLANG_EXE_DIR}" CACHE STRING "")
-    set(BITCODE_DIR "${AMD_DEVICE_LIBS_PREFIX}/lib" CACHE STRING "")
+
+    # Support building with new and old device-lib paths
+    # Previously installed to /opt/rocm/lib
+    # New layout installed to /opt/rocm/amdgcn/bitcode
+    find_path(BC_DIR NAMES "opencl.bc" "opencl.amdgcn.bc"
+      PATHS
+        "${AMD_DEVICE_LIBS_PREFIX}/amdgcn/bitcode"
+        "${AMD_DEVICE_LIBS_PREFIX}/lib/bitcode"
+        "${AMD_DEVICE_LIBS_PREFIX}/lib")
```

Comment:
This probably needs `NO_DEFAULT_PATH` so it only searches the paths in the device libs found.

Reply:
Sure I will add that